test/distributed/org/apache/cassandra/fuzz/topology/AccordTopologyMixupTest.java

```diff
@@ -0,0 +1,397 @@
+    {
+        return AccordState::new;
+    }
+
+    @Override
+    protected void preCheck(Property.StatefulBuilder builder)
+    {
+        // if a failing seed is detected, populate here
+        // Example: builder.withSeed(42L);
+    }
+
+    private static Spec createSchemaSpec(RandomSource rs, Cluster cluster)
+    {
+        TransactionalMode mode = rs.pick(TRANSACTIONAL_MODES);
+        boolean enableMigration = allowsMigration(mode) && rs.nextBoolean();
+        TableMetadata metadata = fromQT(new CassandraGenerators.TableMetadataBuilder()
+                                        .withKeyspaceName(KEYSPACE)
+                                        .withTableKinds(TableMetadata.Kind.REGULAR)
+                                        .withKnownMemtables()
+                                        //TODO (coverage): include "fast_path = 'keyspace'" override
+                                        .withTransactionalMode(enableMigration ? TransactionalMode.off : mode)
+                                        .withoutEmpty()
+                                        .build())
+                                 .next(rs);
+        maybeCreateUDTs(cluster, metadata);
+        String schemaCQL = metadata.toCqlString(false, false, false);
+        logger.info("Creating test table:\n{}", schemaCQL);
+        cluster.schemaChange(schemaCQL);
+        if (enableMigration)
+        {
+            cluster.schemaChange("ALTER TABLE " + metadata + " WITH " + mode.asCqlParam());
+            cluster.get(1).nodetoolResult("consensus_admin", "begin-migration", "--target-protocol", "accord", metadata.keyspace, metadata.name).asserts().success();
+        }
+        return new Spec(mode, enableMigration, metadata);
+    }
+
+    private static BiFunction<RandomSource, State<Spec>, Command<State<Spec>, Void, ?>> cqlOperations(Spec spec)
+    {
+        Gen<Statement> select = (Gen<Statement>) (Gen<?>) fromQT(new ASTGenerators.SelectGenBuilder(spec.metadata).withLimit1().build());
+        Gen<Statement> mutation = (Gen<Statement>) (Gen<?>) fromQT(new ASTGenerators.MutationGenBuilder(spec.metadata).withoutTimestamp().build());
+        Gen<Statement> txn = (Gen<Statement>) (Gen<?>) fromQT(new ASTGenerators.TxnGenBuilder(spec.metadata).build());
+        Map<Gen<Statement>, Integer> operations = new LinkedHashMap<>();
+        operations.put(select, 1);
+        operations.put(mutation, 1);
+        operations.put(txn, 1);
+        Gen<Statement> statementGen = Gens.oneOf(operations);
+        return (rs, state) -> cqlOperation(rs, state, statementGen);
+    }
+
+    private static Command<State<Spec>, Void, ?> cqlOperation(RandomSource rs, State<Spec> state, Gen<Statement> statementGen)
+    {
+        Statement stmt = statementGen.next(rs);
+        String cql;
+        //TODO (usability): are there any transaction_modes that actually need simple mutations/select to be wrapped in a BEGIN TRANSACTION?  If not then this logica can be simplified
```

Comment:
writes are now supposed to be transparent but reads are not... I do think this is also `transaction_mode` explicit... since we are single threaded it would be good to delegate but still need the read side